package/autoyast2.spec

```diff
@@ -108,8 +108,8 @@ installation sources.
 Summary:        YaST2 - Auto Installation Modules
 Group:          System/YaST
 
-# Y2Storage::AutoinstIssues containing section information
-BuildRequires:  yast2-storage-ng >= 4.0.15
+# New API for Y2Storage::PackageHandler and storage features
+BuildRequires:  yast2-storage-ng >= 4.0.95
```

Comment:
I actually fear there is a `Requires` missing for this (sub)package, but I wouldn't dare to change at this point in time how dependencies are organized. If it has been working for so long, I can only guess it's fine.

Reply:
4.2.95, right?

Reply:
Right. Fixed via `git push -f`